Makefile

```diff
@@ -27,12 +27,12 @@ uninstall:
 
 example: reset sharedlib
 	$(CXX) src/pmemkv_example.cc -o pmemkv_example $(USE_PMEMKV) $(CXX_FLAGS)
-	PMEM_IS_PMEM_FORCE=1 ./pmemkv_example
+	LD_LIBRARY_PATH=. PMEM_IS_PMEM_FORCE=1 ./pmemkv_example
```

Comment:
I don't think you want to set it to just '.'.  Shouldn't you add it to existing LD_LIBRARY_PATH? LD_LIBRARY_PATH=.:$LD_LIBRARY_PATH